app/uk/gov/hmrc/softdrinksindustrylevystub/Store.scala

```diff
@@ -42,21 +42,19 @@ object Store {
     def generate(pred: Subscription => Boolean) =
       genSubscription.retryUntil(pred).seeded(sdil)
 
-    val seeded = (sdil.init.last, sdil.last) match {
-      case (_, '0')   => None
-      case ('1', '1') => generate(_.activity.isSmallProducer).map(_.copy(warehouseSites = Nil))
-      case (_, '1')   => generate(_.activity.isSmallProducer)
-      case ('2', '2') => generate(_.activity.isLarge).map(_.copy(warehouseSites = Nil))
```

Comment:
This will break the stubs for anything ending in 22 for example (in addition to others).

Have you tested the application by running it with your new changes?